content/en/docs/tasks/configure-pod-container/subpath-env-expansion.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Config the subpath using environment variables
```

Comment:
Would this be better suited as an addition to the current subpath documentation here:
https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath